modules/deploy/pages/deployment-option/cloud/serverless.adoc

```diff
@@ -0,0 +1,154 @@
+
+For *Go*, run the following command in your terminal:
+
+```
+rpk generate app
+```
+
+== Integrate an existing application
+
+To integrate an existing Kafka application in your Serverless cluster:
+
+. Clone your application.
+. Update your application to use your Serverless cluster.
+. Test your application.
+
+== Supported features
+
+Serverless clusters work with all Kafka clients. For more information, see xref:develop:kafka-clients.adoc[].
+
+Serverless clusters support all major Apache Kafka messages for managing topics, producing/consuming data (including transactions), managing groups, managing offsets, and managing ACLs. User management is available in the Redpanda Cloud UI or with `rpk acl`.
+
+The following Kafka messages are supported:
+
+* `Produce`                     
+* `Fetch`                     
+* `ListOffsets`                 
+* `Metadata`                    
+* `OffsetCommit`                
+* `OffsetFetch`                 
+* `FindCoordinator`             
+* `JoinGroup`                   
+* `Heartbeat`                   
+* `LeaveGroup`                  
+* `SyncGroup`                   
+* `DescribeGroups`              
+* `ListGroups`                  
+* `SASLHandshake`               
+* `ApiVersions`                 
+* `CreateTopics`                
+* `DeleteTopics`                
+* `DeleteRecords`               
+* `InitProducerID`              
+* `OffsetForLeaderEpoch`        
+* `AddPartitionsToTxn`          
+* `AddOffsetsToTxn`             
+* `EndTxn`                      
+* `TxnOffsetCommit`             
+* `DescribeACLs`                
+* `CreateACLs`                  
+* `DeleteACLs`                  
+* `DescribeConfigs`             
+* `AlterConfigs`                
+* `AlterReplicaLogDirs`         
+* `DescribeLogDirs`             
+* `SASLAuthenticate`            
+* `CreatePartitions`            
+* `DeleteGroups`                
+* `IncrementalAlterConfigs`
+
+== Unsupported features
+
+The following features are not yet supported: 
+
+* Redpanda Admin, HTTP Proxy, and Schema Registry APIs are not yet available. You can use an external schema registry.
+* Managed connectors
+* Data transforms
+
+To use these features, to run production workloads with consistently high throughput, or for greater control over your deployment and cluster settings, use Redpanda BYOC or Dedicated clusters. 
+
+== Serverless configurations
+
+Redpanda Serverless is opinionated about Kafka configurations, including the following:
+
+* Automatic topic creation is disabled. 
++
+Some systems expect the Kafka service to automatically create topics when a message is produced to a topic that doesn't exist. Serverless is opinionated about disabling automatic topic creation. Create topics in the Redpanda Cloud UI or with `rpk topic create`.
+
+* Cleanup policy is delete.
```

Comment:
Please remove. Changed in the product

Reply:
@vladoschreiner Can you please confirm: Can this entire section be removed? Or just the last 2 bullets (cleanup policy  & infinite retention)?

Reply:
If we have just that 1 remaining item, can we move it up near Limitations?

Reply:
Yes. Only a single bulletpoint is valid now. Feel free to merge it with Limitations.

Reply:
keeping this as is, at bottom of page, per Vlado